src/imgui_impl_sdl_gl3.cc

```diff
@@ -406,7 +406,9 @@ void ImGui_ImplSdlGL3_Shutdown()
 
 void ImGui_ImplSdlGL3_NewFrameOffscreen(int w, int h)
 {
-    if (!g_FontTexture)
+    ImGuiIO& io = ImGui::GetIO();
+
+    if (!io.Fonts->TexID)
```

Comment:
Do we end up with a duplicate of this in every imgui context now?

Reply:
(That seems suboptimal given we're gong to have lots of them)

Reply:
Yes, presumably. Can be optimized later